Doc/library/dbm.rst

```diff
@@ -51,6 +51,9 @@ the Oracle Berkeley DB.
 .. |flag_n| replace::
    Always create a new, empty database, open for reading and writing.
 
+.. |incompat_note| replace::
+   The file formats created by :mod:`dbm.gnu` and :mod:`dbm.ndbm` are incompatible.
+
 .. function:: open(file, flag='r', mode=0o666)
```

Comment:
```suggestion
.. function:: open(file, flag='r', mode=0o666, /)
```
Should this be added for consistency too?

Reply:
No, because it is not correct. The extension modules that make up the GDBM and NDBM backends are positional only. But the dbm open() "wrapper" is just an ordninary Python function.

Reply:
https://github.com/python/cpython/blob/8278fa2f5625b41be91191d18ee8eeab904a54ff/Lib/dbm/__init__.py#L53

https://github.com/python/cpython/blob/8278fa2f5625b41be91191d18ee8eeab904a54ff/Modules/_dbmmodule.c#L506-L518

https://github.com/python/cpython/blob/8278fa2f5625b41be91191d18ee8eeab904a54ff/Modules/_gdbmmodule.c#L654-L659